Formula/serf.rb

```diff
@@ -18,87 +18,18 @@ class Serf < Formula
   end
 
   depends_on "go" => :build
+  depends_on "govendor" => :build
```

Comment:
Do the deps end up exactly the same with govendor+gox as compared to the current formula in master? If not, may want to revision bump.